test/Microsoft.OpenApi.SmokeTests/Microsoft.OpenApi.SmokeTests.csproj

```diff
@@ -1,11 +1,11 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFrameworks>net48</TargetFrameworks>
+    <TargetFrameworks>net60</TargetFrameworks>
   </PropertyGroup>
 
   <PropertyGroup Condition="'$(Configuration)|$(Platform)'=='Debug|AnyCPU'">
-    <DefineConstants>TRACE;DEBUG;NET48</DefineConstants>
+    <DefineConstants>TRACE;DEBUG;NET60</DefineConstants>
```

Comment:
should that mean we have code-updates to make? if the constant was not used, just remove it